components/Card/Card.css

```diff
@@ -52,11 +52,6 @@
   width: 4rem;
```

Comment:
Instead of using Margin-Top 2%, try to use Margin-Bottom on the .card class.

The newly created cards from your form dissappear behind the footerbar